logstash-codec-nmap.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
   s.metadata = { "logstash_plugin" => "true", "logstash_group" => "codec" }
 
   # Gem dependencies
-  s.add_runtime_dependency "logstash-core-plugin-api", "~> 1.0"
+  s.add_runtime_dependency "logstash-core-plugin-api", "~> 1.60", "<= 2.99"
```

Comment:
Can you also bump the version in this file to 0.0.17: https://github.com/logstash-plugins/logstash-codec-nmap/blob/master/logstash-codec-nmap.gemspec#L4

Also, can you add the infromation to the changelog?

Thanks!